articles/cosmos-db/how-to-setup-rbac.md

```diff
@@ -204,8 +204,8 @@ Create a role named *MyReadOnlyRole* that only contains read actions in a file n
 ```
 
 ```azurecli
-resourceGroupName='<myResourceGroup>'
```

Comment:
I'm not sure why a space is needed between the bash variable and the value. Many Linux examples online do not include a space between the variable and value.